recipes/pyranges/meta.yaml

```diff
@@ -13,8 +13,8 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  number: 0
-  skip: True  # [py!=37]
+  number: 1
+  skip: True
```

Comment:
You're going to want to remove this so it build :)

Reply:
Thanks for reviewing. What should I do to fix it? 

It currently says 

```
  number: 1
  skip: True
```

should it be different?

Reply:
Remove the `skip:` line.

Reply:
Thanks!